beard/clustering/tests/test_blocking.py

```diff
@@ -29,12 +29,16 @@
 from ..wrappers import ScipyHierarchicalClustering
 from ...metrics import paired_f_score
 
+random_state = check_random_state(42)
+X, y = make_blobs(centers=4, shuffle=False, random_state=random_state)
+
+
+def _distance(X_ids):
+    return euclidean_distances(X[X_ids.ravel()])
```

Comment:
If we parallelize it with `multiprocessing`, the `distance` function has to be globally accessible.

To work correctly, it needs an access to `X` variable. As many other tests are using the same data I decided to put the definition here. With this, we also test whether the data is not changed by our library.